basics/concepts.md

```diff
@@ -1,79 +1,82 @@
 
-### **Tenant**
+### Tenant
 
-Pinot supports multi-tenancy. Every Pinot table is associated with a [tenant.](components/cluster/tenant.md) This allows all tables belonging to a particular logical namespace to be grouped under a single tenant name and isolated from other tenants. This isolation between tenants provides different namespaces for applications and teams to prevent sharing tables or schemas. Development teams building applications will never have to operate an independent deployment of Pinot. An organization can operate a single cluster and scale it out as new tenants increase the overall volume of queries. Developers can manage their own schemas and tables without being impacted by any other tenant on a cluster.&#x20;
+Every table is associated with a [_tenant_](components/cluster/tenant.md), or a logical namespace that restricts where the cluster stores and performs query processing on the table. A Pinot tenant takes the form of a text tag in the logical tenant namespace. Physical cluster hardware resources (i.e., [brokers](components/cluster/broker.md) and [servers](components/cluster/server.md)) are also associated with a tenant tag in the common tenant namespace. Tables of a particular tenant tag will only be scheduled for storage and query processing on hardware resources that belong to the same tenant tag. This allows operators of a Pinot cluster to assign certain workloads to certain hardware resources, and to prevent data from separate workloads from being stored or processed on the same physical hardware.
 
-By default, all tables belong to a default tenant named "default". The concept of tenants is very important, as it satisfies the architectural principle of a "database per service/application" without having to operate many independent data stores. Further, tenants will schedule resources so that segments (shards) are able to restrict a table's data to reside only on a specified set of nodes. Similar to the kind of isolation that is ubiquitously used in Linux containers, compute resources in Pinot can be scheduled to prevent resource contention between tenants.
+Multitenancy is an optional feature in Pinot. All tables, brokers, and servers belong to a tenant called _DefaultTenant_ in the absence of explicit tenant configuration.
```

Comment:
```suggestion
By default, all tables, brokers, and servers belong to a tenant called _DefaultTenant_, but you can configure multiple tenants in a Pinot cluster.
```

Reply:
it'd be cool to have a procedure for how to Configure a tenant (curious if configuring a tenant may be part of configuring a table, @matthewhelmke?) 

....the info [here ](https://docs.pinot.apache.org/basics/components/cluster/tenant#tenant-configuration )feels circular